packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/__tests__/configAggregator.spec.js

```diff
@@ -0,0 +1,79 @@
+import configAggregator from '../configAggregator';
+
+test('column config is aggregated correctly without minHeight property', () => {
+    const node = document.createElement('div');
+    node.innerHTML = `<div class="pagebuilder-column myCssColumnClass"
+        data-content-type="column"
+        data-appearance="default"
+        data-background-images='{\\"desktop_image\\":\\"image-desktop.png\\",\\"mobile_image\\":\\"image-mobile.png\\"}'
+        data-element="main"
+        style="justify-content: flex-start;
+            display: flex;
+            flex-direction: column;
+            background-color: rgb(193, 193, 193);
+            background-position: left top;
+            background-size: cover;
+            background-repeat: no-repeat;
+            background-attachment: scroll;
+            text-align: center;
+            border-style: solid;
+            border-color: rgb(0, 0, 0);
+            border-width: 1px;
+            border-radius: 10px;
+            width: calc(33.3333% - 20px);
```

Comment:
```suggestion
            width: calc((100% / 3) - 20px);
```

If you like, you could make this change in the other places this kind of precalculation appears.

Reply:
Updated all occurrences: https://github.com/magento/pwa-studio/pull/1872/commits/51e840ee0742ea494f1d6b57c0b062c8fe83d1c1